src/identity_committer.rs

```diff
@@ -118,15 +119,12 @@ impl IdentityCommitter {
     async fn process_identities(
         database: &Database,
         identity_manager: &(dyn IdentityManager + Send + Sync),
```

Comment:
Probably worth pulling this out into a type synonym in `contracts/mod.rs`. `DynIdentityManager`. Then you can take a `&DynIdentityManager`.

Reply:
I'm assuming this will be monomorphised very soon though?

Reply:
That's true. Leave it.